src/package.json

```diff
@@ -19,8 +19,8 @@
         "tslint": "cd .. && tslint \"**/*.ts\" --config tslint.json --exclude \"**/node_modules/**\"",
         "plugin.link": "npm link && cd ../demo && npm link @essent/nativescript-ng-sentry && cd ../src",
```

Comment:
Please remove old commands, and use new ones from new plugin seed. Also add scripts/build-native.js from new seed also to this plugin. New build command runs this script.